Description: move rom directory to a system path
moved to /usr/share/games/cannonball/rom/
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-04-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/res/config.xml
+++ b/res/config.xml
@@ -5,9 +5,9 @@
 -->
 <data>
     <!-- OutRun Revision B ROM directory -->
-    <rompath>roms/</rompath>
+    <rompath>/usr/share/games/cannonball/rom/</rompath>
     <!-- Path to load required binary files from -->
-    <respath>./res/</respath>
+    <respath>/usr/share/games/cannonball/res/</respath>
     <!-- Path to write XML for scores -->
     <savepath>./</savepath>
     <!-- Use CRC 32 Based ROM Loading (i.e. rely on scanning file contents, not on filenames) -->
